packages/ui-react/src/components/Player/Player.tsx

```diff
@@ -72,7 +72,16 @@ const Player: React.FC<Props> = ({
   const handleComplete = useEventCallback(onComplete);
   const handleUserActive = useEventCallback(onUserActive);
   const handleUserInactive = useEventCallback(onUserInActive);
-  const handleFirstFrame = useEventCallback(onFirstFrame);
+  const handleFirstFrame = useEventCallback(() => {
+    // a11y: auto focus to play/pause button
+    const playerContainer = playerRef.current?.getContainer();
+    const playPauseButton = playerContainer?.querySelector(
```

Comment:
This seems to be the most logical event because a) the player is ready for playback, and b) the player controls are visible.

The best alternative would be the ready event. However, this won't work well when using the InlinePlayer because it will disrupt the navigation by focusing the play button when loading the page.

Do you think of an alternative event?

Reply:
Maybe using `playerInstance` in the `handleFirstFrame` method (PlayerContainer)?